### DISCUSSION TITLE
GitLens joins forces with GitKraken!

### DISCUSSION THREAD

**Discussion by eamodio** [gitkraken/vscode-gitlens#1676] [Feedback]
I’m extremely excited to share some big news &mdash; GitLens and I are joining GitKraken! I am incredibly excited about this! GitLens will have a very bright future and will now finally get the attention it deserves — the full-time focus of myself and the incredible GitKraken team. Stay tuned for what's coming next!

You can read more about it in our [blog post](https://www.gitkraken.com/blog/gitkraken-acquires-gitlens-for-visual-studio-code) or on the [FAQ page](https://gitlens.amod.io/#faq).

I also want to say a HUGE thank you to all my sponsors and everyone who has contributed (code, suggestions, issues, promotion, etc, etc) to making GitLens the incredible product and success it is today!

I really can't thank you all enough! 🙏❤️

**Comment by finn-wa** [comment]
Congrats! GitLens is a lifesaver and I'm excited to see what you'll do at GitKraken :smile:

**Reply by eamodio** [reply]
Thank you!

**Comment by mahaveerbana** [comment]
really it is very helpful
